content/strategy-goals/strategy/code-graph/code-insights/index.md

```diff
@@ -153,10 +155,9 @@ The next six months are focused on building a product that can quickly prove out
 
 ### Goals
 
-#### FY22Q3
+#### FY22q4
```

Comment:
Given the quarter ends in 13 days, maybe just remove this section in this update. Up to you though.

Reply:
Fair point! Since I have a strong hypothesis but haven't shared with the team what our FY23Q1 OKR may be yet, I will leave this in and expect to replace it in next month's update.